<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>firstname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>lastname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>age</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>info1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>info2</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="readFileButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>read file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menug">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpenCsv"/>
   </widget>
   <addaction name="menug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenCsv">
   <property name="text">
    <string>open csv</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
